charts/crljanitor/templates/deployment.yaml

```
@@ -112,6 +112,21 @@ spec:
             secretKeyRef:
               name: crl-janitor-admin-user-list
               key: admin-users
+        - name: AZURE_MANAGED_APP_CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              name: crl-janitor-azure-managed-app-creds
+              key: client-id
+        - name: AZURE_MANAGED_APP_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: crl-janitor-azure-managed-app-creds
+              key: client-secret
+        - name: AZURE_MANAGED_APP_TENANT_ID
+          valueFrom:
+            secretKeyRef:
+              name: crl-janitor-azure-managed-app-creds
+              key: tenant-id
```

```suggestion
       {{- if .Values.azureManagedApp.enabled }}
        - name: AZURE_MANAGED_APP_CLIENT_ID
          valueFrom:
            secretKeyRef:
              name: crl-janitor-azure-managed-app-creds
              key: client-id
        - name: AZURE_MANAGED_APP_CLIENT_SECRET
          valueFrom:
            secretKeyRef:
              name: crl-janitor-azure-managed-app-creds
              key: client-secret
        - name: AZURE_MANAGED_APP_TENANT_ID
          valueFrom:
            secretKeyRef:
              name: crl-janitor-azure-managed-app-creds
              key: tenant-id
        {{- end  }}
```

I'd reccomend wrapping this and the associated secretDefinition block this in a conditional and setting a default in `values.yaml` if we don't want this universally enabled in all environments yet. If we do want this enabled everywhere then feel free to disregard.

---

Good call.. `crljanitor` is a bit weird because there is no "live" environment -- the "production" instance is the `tools` personal environment: https://github.com/broadinstitute/terra-helmfile/blob/f01ea9932d1c400c36fb455896aa648d55a808c5/environments/personal/tools.yaml

But maybe it's a good idea to only enable Azure in the `tools` env..

---

Yeah my main concern here is potentially breaking any personal environments that may be using janitor. Although I'm not sure how much use those actually see these days anyways.